src/pudl/metadata/resources/eia860.py

```diff
@@ -546,7 +546,7 @@
     },
     "boiler_emissions_control_equipment_assn_eia860": {
         "description": (
-            """A table that links EIA boiler id to emissions control IDs for nox, so2,
+            """A table that links EIA boiler ID to emissions control IDs for nox, so2,
```

Comment:
Should be NOx and SO2, and "EIA boiler IDs".
Re: the 1:many relationship, is this really 1:many or is this actually an m:m relationship?

Reply:
It's m:m, you're right!